src/pudl/analysis/state_demand.py

```diff
@@ -586,6 +141,25 @@ def total_state_sales_eia861(
     return df[["state_id_fips", "year", "demand_mwh"]]
 
 
+@asset(
+    io_manager_key="parquet_io_manager",
+    compute_kind="Python",
+    required_resource_keys={"dataset_settings"},
+)
+def out_ferc714__hourly_planning_area_demand(
+    context,
+    core_ferc714__hourly_planning_area_demand: pd.DataFrame,
+) -> pd.DataFrame:
+    """Impute hourly demand on core_ferc714__hourly_planning_area_demand table."""
+    df = pudl.analysis.timeseries_cleaning.impute_timeseries(
+        core_ferc714__hourly_planning_area_demand,
+        years=context.resources.dataset_settings.ferc714.years,
+        value_col="demand_mwh",
+        id_col="respondent_id_ferc714",
+    )
+    return df.drop_duplicates(subset=["respondent_id_ferc714", "datetime_utc"])
+
+
```

Comment:
This and the other `out_ferc714` assets feel like they would be more intuitively found in the `pudl.output.ferc714` module now, since they don't just pertain to the State Demand outputs.

Reply:
Moved